runtime/compiler/control/CompilationThread.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2000, 2021 IBM Corp. and others
+ * Copyright (c) 2000, 2022 IBM Corp. and others
```

Comment:
My intuition is that these should be updated in the first commit that touches each file, but I'm not sure how it's usually handled, so maybe the separate commit is fine

Reply:
Ideally yeah, but it's not unprecedented to have a separate commit to update the copyrights, especially when it would be tedious to go back and updated all the various commits.